subprojects/language-native/src/main/java/org/gradle/language/swift/tasks/CreateSwiftBundle.java

```diff
@@ -39,20 +44,20 @@
 public class CreateSwiftBundle extends DefaultTask {
     private final RegularFileVar informationFile;
     private final RegularFileVar executableFile;
-    private final DirectoryVar outputDir;
+    private final RegularFileVar outputDir;
```

Comment:
OK... so it looks like it's a file now.  Should this be called something else?

Reply:
See the other comment on this issue.